src/main/java/com/glovo/ttglovo/securityManagement/appuser/AppUserInfoController.java

```diff
@@ -0,0 +1,27 @@
+package com.glovo.ttglovo.securityManagement.appuser;
+
+import lombok.AllArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@AllArgsConstructor
+public class AppUserInfoController {
+
+    private final AppUserService appUserService;
+
+    @GetMapping("/client")
+    public String currentUser() {
+        //TODO atentie user e email!!!
+
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        AppUser user = (AppUser) appUserService.loadUserByUsername((String) authentication.getPrincipal());
+        return user.getUsername() + "\n" + user.getAppUserRole();
```

Comment:
I know it looks nicer, but if possible return a single line strings. The line endings are a nuisance to process.